paasta_tools/cleanup_chronos_jobs.py

```diff
@@ -28,9 +28,11 @@
 import argparse
 import sys
 
+import pysensu_yelp
 import service_configuration_lib
 
 from paasta_tools import chronos_tools
+from paasta_tools.check_chronos_jobs import send_event_to_sensu
```

Comment:
Instead of doing this, can you move ``paasta_tools.check_chronos_jobs.send_event_to_sensu`` to ``paasta_tools.chronos_tools`` or even ``paasta_tools.utils`` if it is generic enough?